### DISCUSSION TITLE
How to make the QEditor border thicker?

### DISCUSSION THREAD

**Discussion by wsw70** [quasarframework/quasar#9579] [General - Components / Directives / etc] [answered]
I would like the border of my QEditor to be 3px instead of the default 1px. How can I do that?

I added the styling 

```
.q-editor {
  border-width: 3px !important;
}
```

but it seems quite forceful - maybe there is a more standard way to do that?

**Comment by ontwikkelfabriek** [comment, marked as answer]
@wsw70 

Here you go:
https://codepen.io/ontwikkelfabriek/pen/JjWvmNJ

**Reply by wsw70** [reply]
Thank you! I did not realize that `.q-editor.q-editor--default` was the call to go after.

**Reply by ontwikkelfabriek** [reply]
@wsw70 

Chrome inspector is how I figured it out.
